controller_manager_tests/test/controller_manager_scripts.py

```diff
@@ -107,6 +112,32 @@ def test_scripts(self):
 
         self.assertEqual(run_cm('reload-libraries'), reload_response)
 
+        # controller_group
+        # list
+        self.assertEqual(
+            run_cg('list'),
+            'Number of groups: 3\n  group1\n      my_controller1\n      my_controller3\n'
```

Comment:
Could you also move group1 to its own line, please? It'd be easier to read.

Reply:
Sure